### DISCUSSION TITLE
Cannot open file with sqlite

### DISCUSSION THREAD

**Discussion by kaleidawave** [wasmerio/wasmer#3381] [General]
I am trying the sqlite project with Wasmer. However any interactions with the file system seem to fail:
```
> wasmer run sqlite test.sl3
SQLite version 3.26.0 2018-12-01 12:34:55
Enter ".help" for usage hints.
sqlite> create table test (x int);
Error: unable to open database "test.sl3": unable to open database file
```
Any ideas, I thought WASI enabled file access...? (on windows btw)

**Comment by ptitSeb** [comment]
You probably need to use `mapdir` to map current folder

**Reply by kaleidawave** [reply]
Can you elaborate, is that with the `wasmer run --mapdir` option...? Having the same for `wasmer run python .\hello.py`, which outputs `cannot open '.\hello.py', [Errno 76] Capabilities insufficient`. 

Is there a flag like in Wasmer like Deno's `--allow-read`?

**Comment by sigmaSd** [comment]
I'm also curious how to pass permissions to wasm program, searching for `Capabilities` in the docs doesn't yield anything

**Reply by sigmaSd** [reply]
I think its just not implemented yet, since wasmtime has the same error, but can pass permissions

**Reply by sigmaSd** [reply]
Turns out its possible `wasmer run --map-dir ".:." ./cp.wasm -- ./src/foo ./dest/bar`
